src/platform/julius.c

```diff
@@ -324,6 +343,9 @@ static int init_sdl(void)
         SDL_LogError(SDL_LOG_CATEGORY_APPLICATION, "Could not initialize SDL: %s", SDL_GetError());
         return 0;
     }
+#if SDL_VERSION_ATLEAST(2, 0, 3)
```

Comment:
Does not compile on 2.0.3: min sdl version required is 2.0.4

Reply:
Oops, I misinterpreted the SDL manual.